subprojects/platform-play/platform-play.gradle

```diff
@@ -28,3 +41,15 @@ useTestFixtures(project: ':platformBase')
 
 useClassycle()
 strictCompile()
+
+task integTestPrepare(type: org.gradle.testing.IntegrationTest) {
```

Comment:
I was wondering too. However, I tend to prefer separate source sets now, after seeing how include/excludes quickly becomes a mess in the Groovy build...